File: DynamoDB-SDK-Examples/golang/WorkingWithTables/disableStreams/disableStreams.go
package main

import (
    "github.com/aws/aws-sdk-go/aws"
    "github.com/aws/aws-sdk-go/aws/session"
    "github.com/aws/aws-sdk-go/service/dynamodb"
    "fmt"
)

var tableName = "Music"
var awsRegion = "us-west-2"

func getSession() (*session.Session) {
    sess := session.Must(session.NewSessionWithOptions(session.Options{
        SharedConfigState: session.SharedConfigEnable,
        // Provide SDK Config options, such as Region and Endpoint
        Config: aws.Config{
            Region: aws.String(awsRegion),
        },
    }))

    return sess
}

func enableStreams() error {
    dynamoDBClient := dynamodb.New(getSession())

    _, err := dynamoDBClient.UpdateTable(&dynamodb.UpdateTableInput{
        StreamSpecification: &dynamodb.StreamSpecification{
            StreamEnabled:  aws.Bool(false),
        },
        TableName: aws.String(tableName),
    })

    if err != nil {
        fmt.Println("Error disabling streams", err)
        return err
    }

    return nil
}

func main() {
    fmt.Println("Diasbling streams in table ...")
    enableStreams()
    fmt.Println("Finished ...")
}

File: DynamoDB-SDK-Examples/node.js/WorkingWithTables/disable_streams.js
// This code disables DynamoDB Streams on the target table.

const {DynamoDBClient, UpdateTableCommand } = require('@aws-sdk/client-dynamodb');

const REGION = "us-west-2";
const TableName = "Music";

const dbclient = new DynamoDBClient({ region: REGION });

async function disableStreams() {
    const params = {
        TableName: TableName,
        StreamSpecification: {
            StreamEnabled: false,
        },
    };
    return await dbclient.send( new UpdateTableCommand(params));
}

disableStreams()
    .then((data) => console.log(data))
    .catch((error) => console.log("An error occured while disabling streams:" + ' ' + error.message ));
